frontend/src/redux/auth/action-types.ts

```diff
@@ -0,0 +1,5 @@
+import { Auth, ReduxAction } from "../../types"
+
+export const UPDATE_AUTH = "UPDATE_AUTH"
```

Comment:
As we discussed on Friday, I think there's a valid argument for having two actions here: `RECEIVED_LOGIN` and `LOGOUT`, only the former of which takes a payload. Just voicing that, definitely no need to change.

Also, something to consider: seems like the standard action naming is moving in the direction of `auth/update` rather than `UPDATE_AUTH`. But, we can leave that for our conversation around migrating to redux toolkit, since they do that by default.

Reply:
I agree w/ the actions actually, this pattern was honestly just a quicker way of writing the code, as you can copy/paste the caps lock version more quickly. As far as the multiple calls, I'm still not totally sold, and would like to see a good example of what you had in mind. The code in Stash made sense, but to me it became very verbose across multiple files. That could have just been a side-effect of how that app was written however, and I don't wanna say that is always what this pattern would produce.